product_pricelist_direct_print/wizards/product_pricelist_print.py

```diff
@@ -66,6 +66,7 @@ class ProductPricelistPrint(models.TransientModel):
         " the last X ordered products will be obtained for the report."
     )
     summary = fields.Text()
+    print_child_categories = fields.Boolean(default=True)
```

Comment:
print_child_categories = fields.Boolean(compute="_compute_print_child_categories", readonly=False)
@api.depends("categ_ids")
def _compute_print_child_categories(self):
    for rec in self:
         rec.print_child_categories = True if rec.categ_ids else False